src/frontend/js/api/joanie.ts

```diff
@@ -297,12 +297,11 @@ const API = (): Joanie.API => {
           }).then(checkStatus),
       },
       orders: {
-        abort: async ({ id, payment_id }) => {
+        abort: async ({ id, payment_id }) =>
           fetchWithJWT(ROUTES.user.orders.abort.replace(':id', id), {
             method: 'POST',
             body: payment_id ? JSON.stringify({ payment_id }) : undefined,
-          }).then(checkStatus);
-        },
+          }).then(checkStatus),
```

Comment:
What do you think to use an explicit return statement ?

```suggestion
        abort: async ({ id, payment_id }) => {
          return fetchWithJWT(ROUTES.user.orders.abort.replace(':id', id), {
            method: 'POST',
            body: payment_id ? JSON.stringify({ payment_id }) : undefined,
          }).then(checkStatus);
        },
```